### ISSUE TITLE
[JENKINS-17224] HudsonTestCase cannot be run in parallel

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-14#2000]
<p>The temporary folder for the Jenkins installation is always the same on Windows. Therefore, running multiple JUnit tests in parallel does not work.</p>

<p>Solution: use some random prefix for the folders</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=gruberrolandfiducia">gruberrolandfiducia</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-17224" target="_blank">HudsonTestCase cannot be run in parallel</a></i></summary>
<i><ul>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>resolution</b>: Won't Do
<li><b>resolved</b>: 2017-04-04T16:36:04+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>The temporary folder for the Jenkins installation is always the same on Windows. Therefore, running multiple JUnit tests in parallel does not work.</p>

<p>Solution: use some random prefix for the folders</p></pre>
</details>
</details>
